doc/changelog.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+2.0.0a3
+-------
+
+Fixes:
+
+* Jinja templates can now be loaded in StrictUndefined mode.
```

Comment:
I wasn't sure if 1) this change warranted an entry,
2) this was the correct version header

Reply:
This looks great - that'll be the next version, so perfect :)